app/scripts/modules/docker/src/image/DockerImageAndTagSelector.tsx

```diff
@@ -38,9 +38,9 @@ export interface IDockerImageAndTagSelectorProps {
 
 export interface IDockerImageAndTagSelectorState {
   accountOptions: Array<Option<string>>;
+  switchedManualWarning: string;
```

Comment:
If this were optional, you could save a line of code and reduce our bundles sizes considerably.

Reply:
True.